lightning/cookie.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from datetime import datetime
+from lightning_adapter.models import Cookie as RawCookie
+
+
+class Cookie(dict):
+
+    def __init__(self, *, name: str, value: str, path: str = None,
+                 domain: str = None, secure_only: bool = False,
+                 http_only: bool = False, expires: datetime.timestamp = None,
+                 same_site_policy: str = None):
```

Comment:
Use SameSitePolicy enum from generated code if exists.

Reply:
In docs it is described as str

Reply:
Ok